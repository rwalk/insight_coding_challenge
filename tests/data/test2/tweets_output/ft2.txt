10
9.5
9
9.5
10
11.5
10
11.5
10
11.5
10
10.0
10
10.5
10
10.5
10
10.5
11
10.5
10
10.0
10
10.0
10
10.0
10
10.5
11
10.5
11
11.0
11
12.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
13.0
13
12.0
11
12.0
11
11.0
11
11.0
11
11.0
11
11.0
11
12.0
13
12.0
11
12.0
13
12.0
13
12.0
11
11.0
11
11.0
11
11.0
11
11.0
11
11.0
11
11.0
11
11.0
11
11.0
