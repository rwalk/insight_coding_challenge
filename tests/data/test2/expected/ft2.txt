10
9
9
16
13
18
10
17
10
17
10
5
13
11
0
19
9
16
17
7
9
10
10
18
16
7
13
19
11
9
13
17
13
17
17
18
13
13
13
15
9
9
16
9
17
13
7
11
9
16
5
16
16
7
13
19
7
0
9
17
8
10
8
9
9
16
7
7
16
5
17
7
8
17
13
13
17
10
8
18
18
9
17
9
9
0
17
5
11
18
10
8
9
16
17
9
13
18
10
15
